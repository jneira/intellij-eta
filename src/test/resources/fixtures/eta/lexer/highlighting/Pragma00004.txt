ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Pragma00004')
WHITE_SPACE ('  ')
ITinline_prag ('{-#')
ITline_prag (' WARNING "This is an unstable interface." ')
ITline_prag ('#-}')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
