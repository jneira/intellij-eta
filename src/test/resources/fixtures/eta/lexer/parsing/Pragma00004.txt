ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Pragma00004')
WHITE_SPACE ('  ')
ITwarning_prag ('{-# WARNING')
WHITE_SPACE (' ')
ITstring ('"This is an unstable interface."')
WHITE_SPACE (' ')
ITclose_prag ('#-}')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvocurly ('')