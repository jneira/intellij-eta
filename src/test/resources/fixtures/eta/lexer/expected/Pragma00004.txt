ITmodule ('module')
WHITE_SPACE ('')
ITconid (' Pragma0000')
WHITE_SPACE ('4 ')
ITwarning_prag (' {-# WARNIN')
WHITE_SPACE ('G')
ITstring (' "This is an unstable interface.')
WHITE_SPACE ('"')
ITclose_prag (' #-')
WHITE_SPACE ('}')
ITwhere (' wher')
WHITE_SPACE ('e\n\n')
ITvocurly ('')